zio-http/shared/src/main/scala/zio/http/endpoint/Endpoint.scala

```diff
@@ -609,10 +609,10 @@ final case class Endpoint[PathInput, Input, Err, Output, Auth <: AuthType](
     Endpoint(
       route,
       input,
-      output = ((HttpCodec.content[Output2](mediaType) ++ StatusCodec.status(status)) ?? doc) | self.output,
+      output = (HttpCodec.content[Output2](mediaType) ++ StatusCodec.status(status) ?? doc) | self.output,
```

Comment:
As far as I can tell, this has either no effect on the codec or applies the doc only to status. In either way, I'd prefer to keep this. Unless you can tell me why that would turn tests read.

Reply:
You're right, this and the other change in `Endpoint` seem to have no effect on the codec - the tests are still green with the old code.
I think I was just playing around, trying to find the culprit, and thought this was it 😅